### ISSUE TITLE
Percent Loaded

### ISSUE THREAD

**Issue by kyoukhana** [VJAI/musquito#7]
I don't see in the docs some sample code that when loading a single mp3 or multiple sounds it would show some kind of indication or percentage loaded.  Lets say you loading  3 files at once can this library load each file in sequence once the previous files is loaded.

**Comment by VJAI** [comment]
I can provide you a new event called `load-progress` that can give info about the percentage loaded for single and multiple files. I'm not sure i can support sequential loading. I'll update you in a day.

**Comment by VJAI** [comment]
This feature is now available in 1.1.5.

```js
const buzz = $buzz({
    url: 'sounds/bg.mp3',
    onloadprogress: function(percentage) {
       console.log(percentage);
    }
});
```

```js
$buzz.setup();
$buzz.load('sounds/bg1.mp3', 'sounds/bg2.mp3, function(evtData) {
    console.log(evtData.url);
    console.log(evtData.percentageDownloaded);
});
```

PLEASE GIVE A STAR TO REPO THAT'LL MOTIVATE ME TO DO MORE IMPROVENTS!
